site/docs/foundations/shadow.mdx

```diff
@@ -19,4 +26,48 @@ There are six shadow elevations to choose from.
 | `--salt-shadow-400` | 0           | 6           | 10        |
 | `--salt-shadow-500` | 0           | 12          | 40        |
 
+## Scrollable area: Shadow-100
```

Comment:
It feels weird that this whole page is about shadow ramps then starts immediately referring them to by use-case or semantics. Should the order of the heading be swapped?

Reply:
Good point, will change this